app/src/main/res/menu/menu_plant_detail.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This line should be below the copyright.

Reply:
1) i copied from existing menu.xml in the project and edited the item info only.
2) if you move this line below copyright, AS will have an error 'Xml declaration should precede all document content'. you wont be able to compile it

Reply:
My mistake - you're correct.  No action needed here.